.github/tests/application/Makefile

```
@@ -0,0 +1,28 @@
+build:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)